AEPTarget/Sources/Target.swift

```diff
@@ -638,8 +638,10 @@ public class Target: NSObject, Extension {
 
         let response = TargetDeliveryResponse(responseJson: responseDict)
 
-        if let connectionResponseCode = connection.responseCode,
-           connectionResponseCode != 200 {
+        if
+            let connectionResponseCode = connection.responseCode,
+           connectionResponseCode != 200
```

Comment:
were the whitespace changes here intentional?

Reply:
This was to resolve the conflicting linting & formatting rules, fixed it.